scripts/signedbuildskadi.sh

```diff
@@ -44,6 +45,7 @@ else
     NGINX_PASSWORD="skadi"
     GRAFANA_USER=$NGINX_USER
     GRAFANA_PASSWORD=$NGINX_PASSWORD
+    HEAP_SIZE=$HEAP_SIZE
```

Comment:
I think this will need to be passed as an env variable with a default option if no env variable found.

for example
HEAP_SIZE=${HEAP_SIZE:-1g)

Reply:
should be corrected in lastest commit